src/components/Visualization/styles/Visualization.module.css

```diff
@@ -83,10 +83,10 @@
     padding: 8px 4px 6px 8px;
 }
 .dataTable .headerCell.sizeNormal {
-    padding: 4px 2px 2px 6px;
+    padding: 4px 4px 2px 6px;
```

Comment:
what are all of these changes and why aren't they mentioned in the PR description?
Screenshots detailing this would be great as well. A ticket number would help as well.

Reply:
Description updated.